checkr/src/app.ts

```diff
@@ -0,0 +1,27 @@
+import express from "express";
+import bodyParser from "body-parser";
+import { syncModels } from "./utils/db";
+import userRoutes from "./routes/user";
+import candidateRoutes from "./routes/candidate";
+import cors from "cors";
+import { config } from "dotenv";
+import { errorMiddleware } from "./middlewares/error-middleware";
+
+config();
+
+const app = express();
+
+app.use(cors());
+
+app.use(bodyParser.json());
```

Comment:
we do not need body parser library, with express 4.16+

Reply:
Okay. Noted.
Will remove it.